src/3D/Camera.h

```diff
@@ -45,6 +37,11 @@ class Camera
 	[[nodiscard]] const glm::mat4& GetProjectionMatrix() const { return _projectionMatrix; }
 	[[nodiscard]] virtual glm::mat4 GetViewProjectionMatrix() const;
 
+	[[nodiscard]] const std::optional<ecs::components::Transform> RaycastCamToLand();
+	[[nodiscard]] const std::optional<ecs::components::Transform> RaycastMouseToLand();
```

Comment:
Move const to end of the line.
Also these functions aren't called, can you provide an example of how they would be used?
Trying to avoid adding dead code

Reply:
`RaycastCamToLand()` is a hangover from when I had the camera rotating around the center of the camera view for orbit rotation, it's now been deleted.

`RaycastMouseToLand()` is used to determine where a camera flight path should end and is used twice further down.